data/language/en-GB.txt

```diff
@@ -3677,6 +3677,7 @@ STR_6360    :{SMALLFONT}{BLACK}{COMMA32}
 STR_6361    :Enable lighting effects on rides (experimental)
 STR_6362    :{SMALLFONT}{BLACK}If enabled, vehicles for tracked rides will be lit up at night.
 STR_6363    :Copied text to clipboard
+STR_6368    :For compatability reasons, it is not recommended to run OpenRCT2 with WINE. Instead, Linux is supported.
```

Comment:
A few comments:
- The number should be the increment from the previous
- Typo fix and a change on the last sentence which I think makes it clearer.
```suggestion
STR_6364    :For compatibility reasons, it is not recommended to run OpenRCT2 with WINE. Linux is natively supported.
```